src/components/pattern-form/pattern-form.js

```diff
@@ -44,7 +44,9 @@ const PatternForm = ({ pattern, id }) => {
   } = usePatternHandlers();
   const languageTabs =
     languages.length > 0
-      ? languages.map((lang, index) => <Tab label={lang} key={lang} />)
+      ? languages.map((lang, index) => (
```

Comment:
You do not use the `index` argument, there is no need to provide it

Reply:
fixed